.gitignore

```diff
@@ -24,3 +24,4 @@ dist-ssr
 *.sw?
 # environment variables
 .env
+*.toml
```

Comment:
```suggestion
```
This should not be added in the .gitignore, since Cargo.toml is used for dependency management